design/src/components/Form.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
Should these have a semicolon at the end of the line? Also applies to next line below.

Reply:
Same in `Footer.jsx`, should the lines end with semicolons as they're used in other places? Not sure about the code style so feel free to disregard this, if you've opted out of semicolons.